sequences/voxels.yaml

```diff
@@ -65,9 +65,21 @@ naflak:
 flata: #Unused
 	idle:
 
-fv: #Has different turrets per unit thats inside,default is patriot
+fv:
 	idle:
 	turret: fvtur
+	turret-mg: fvtur1
+	turret-gren: fvtur2
+	turret-advm: fvtur3
+	turret-sniper: fvtur4
+	turret-chrono: fvtur5
+	turret-flak: fvtur6
+	turret-tesla: fvtur7
+	turret-demo: fvtur8
+	turret-deso: fvtur9
+	turret-psi: fvtur10
+	turret-hero: fvtur11
+	turret-engi: fvtur12
```

Comment:
Having them here doesn't harm but they are also defined like that under `WithVoxelTurret:`.